src/components/pages/PageBase.styled.jsx

```diff
@@ -58,7 +58,16 @@ export const LinkWrapper = styled.div`
 export const PageContentWrapper = styled.section`
   padding: 2rem 4rem;
   flex: 1;
+  min-height: calc(85vh - 50px);
```

Comment:
Do you mind sharing more about this line was needed?

Reply:
I needed a way to get the footer to stick to the bottom of the page regardless of how much content existed - screenshot below as an example of what the footer looks like without  `min-height: calc(85vh - 50px);` and without content that fills the page.  Setting `min-height: 85vh` on `PageContentWrapper` seemed to solve that by making sure that the content section filled enough space regardless of how much content there actually was.

The `calc()` was intended to account for manually setting the footer height to 50px as in [this example](https://css-tricks.com/couple-takes-sticky-footer/#aa-there-is-calc-reduced-height-wrappers) but after some adjusting I don't think that was necessary. I'll add a commit that removes the min-height calculation as I don't think it's doing anything useful in this context.

<img width="1334" alt="Screenshot 2023-05-31 at 7 41 16 AM" src="https://github.com/deardurham/dear-petition/assets/113562625/fccca95f-a8e3-4d3f-aafd-a6877c9c9dc8">

Reply:
Can you use flex instead? The wrapper div would be `flex flex-column` with the main content div having `flex-1`. This will result in the main page content taking up as much vertical space as possible with the footer taking as little space as possible.

Reply:
Helpful guide to flex: https://css-tricks.com/snippets/css/a-guide-to-flexbox/#

Reply:
yep, easy change! Thanks for the tip